tests/core.py

```diff
@@ -42,7 +42,7 @@
 DEFAULT_DATE_ISO = DEFAULT_DATE.isoformat()
 DEFAULT_DATE_DS = DEFAULT_DATE_ISO[:10]
 TEST_DAG_ID = 'unit_tests'
-configuration.test_mode()
+
```

Comment:
configuration.test_mode() is used through out core.py. Should these all be replaced?